src/Microsoft.Azure.AppConfiguration.AspNetCore/Microsoft.Azure.AppConfiguration.AspNetCore.csproj

```diff
@@ -14,6 +14,11 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <PackageReference Include="Azure.Messaging.EventGrid" Version="4.7.0" />
+    <PackageReference Include="Microsoft.Extensions.Configuration.AzureAppConfiguration" Version="4.5.0" />
```

Comment:
><PackageReference Include="Microsoft.Extensions.Configuration.AzureAppConfiguration" Version="4.5.0" /> [](http://example.com/codeflow?start=4&length=103)

Remove reference to 4.5.0 Microsoft.Extensions.Configuration.AzureAppConfiguration nuget package. We now refer to the project directly (see ProjectReference below this).